### ISSUE TITLE
ValueError: invalid interpolation syntax in '...' at position ...

### ISSUE THREAD

**Issue by peteruithoven** [Chadsr/NordVPN-NetworkManager#29]
Hi folks,
Thanks for working on this! 

Looks like the script can't handle some passwords, Trying mine I got the following error:
```
ValueError: invalid interpolation syntax in '...' at position 13
```
At position 13 there was a `%`. When I tried it without the `%` the password was accepted.

**Comment by Chadsr** [comment]
Hi,

Looks like the package I use for parsing the configuration files was allowing parsing of Python string formatting by default. I've just disabled it and tried a few test strings with Pythonic formatting related characters and it seems fixed.

The fix is in the master branch (#30) and will be in the next release.

Thanks for pointing this out!

**Comment by peteruithoven** [comment]
Awesome, thank you!

**Comment by Chadsr** [comment]
Just to let you know, the fix has been released and is available in v0.0a11 from PyPi.
